### PR TITLE
Allow a mailto to support email

### LINKED ISSUE #3919: Support button: Create support request
The green support button in the footer of the UI could do the following:

Condition:

* An administrator is logged in
* The administrator has sensible rights (to ensure, that it is not only an operator) like writing config. Alternatively, we could add  a new admin policy like "open support requests".
* A subscription is installed and valid.

Action:

Open an email program (mailto:) to the issued email address specified in the subscription.

### PR DESCRIPTION
**Pull request by cornelinux**
If the setup has a subscription, the green Support button will create a mailto to the support email address from the subscription.

In case:
- The logged in user is an admin
- and has the right to configwrite.

Working on #3919

### CODE REVIEW

File: privacyidea/static/components/login/controllers/loginControllers.js [hunk 1]
```diff
@@ -417,7 +417,10 @@ angular.module("privacyideaApp")
             $scope.backend_debug_passwords = data.result.value.debug_passwords;
             $scope.privacyideaVersionNumber = data.versionnumber;
             var lang = gettextCatalog.getCurrentLanguage();
-            $scope.privacyideaSupportLink = "https://netknights.it/" + lang + "/support-link-" + data.result.value.role;
+            if (data.result.value.hasOwnProperty("supportmail"))
```
Comment (nilsbehlen) [inline privacyidea/static/components/login/controllers/loginControllers.js:420]: please use `{ }` even for one-liners to keep it consistent

File: tests/test_api_lib_policy.py [hunk 2]
```diff
@@ -4313,6 +4314,52 @@ def test_10_get_webui_settings_admin_dashboard(self):
 
         delete_policy("pol_dashboard")
 
+    def test_11_get_webui_settings_support_link(self):
+        # Test the link to the support
+        self.setUp_user_realms()
+
+        # The request with an OTP value and a PIN of a user, who has not
```
Comment (nilsbehlen) [inline tests/test_api_lib_policy.py:4321]: who has *no* token

**Review by nilsbehlen — CHANGES_REQUESTED** [review]

File: privacyidea/static/components/login/controllers/loginControllers.js (hunk 1 above)
Comment (cornelinux) [reply privacyidea/static/components/login/controllers/loginControllers.js:420]: Will do

File: tests/test_api_lib_policy.py (hunk 2 above)
Comment (cornelinux) [reply tests/test_api_lib_policy.py:4321]: Will do all occurances

Comment (cornelinux) [reply tests/test_api_lib_policy.py]: done in 849b6aec9

File: privacyidea/static/components/login/controllers/loginControllers.js (hunk 1 above)
Comment (cornelinux) [reply privacyidea/static/components/login/controllers/loginControllers.js]: done in 849b6aec9

**Review by nilsbehlen — APPROVED** [review]

**Pull request merged**